package.json

```diff
@@ -64,7 +65,7 @@
     "recursive-readdir": "^2.1.0",
     "rimraf": "^2.5.0",
     "solid-namespace": "^0.1.0",
-    "solid-permissions": "^0.5.2",
+    "solid-permissions": "solid/solid-permissions#feature-group-acl",
```

Comment:
Should be updated to release version.

(Note: in the future, you can just put the next release version here, and use `npm ln` locally.)